druid/src/event.rs

```diff
@@ -131,6 +137,17 @@ pub enum Event {
     Command(Command),
 }
 
+/// Application life cycle events.
+#[derive(Debug, Clone, Copy)]
+pub enum LifeCycle {
+    /// Sent to the root widget of a window when that window is first instantiated.
```

Comment:
This comment doesn't seem to quite match the event propagation, I believe it's delivered to all widgets (which I think is more useful).